tests/src/Interop/IJW/ijwhostmock/MscoreeExports.def

```diff
@@ -0,0 +1,3 @@
+EXPORTS
```

Comment:
Can we use `__declspec(dllexport)`?

Reply:
We need to make sure we match the names on all platforms, and the stdcall mangling breaks that invariant. For the `GetTokenForVTableEntry` entrypoint we could leave it and have CoreCLR look up the mangled name on x86 and the nonmangled name on non-x86 if we want, but we need to have this file anyway for `_CorDllMain` since we need to match the old entrypoint name for our fake mscoree.

Reply:
You can control that as follows:

```
#if defined _X86_
#pragma comment(linker, "/export:FuncWithStdCall=_FuncWithStdCall@4")
#endif
```

Reply:
I'll do that then.